packages/showcases/svelte/stories/components/selection-controls/VtmnTooltip/VtmnTooltip.stories.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import { VtmnTooltip } from '@vtmn/svelte';
+  import { parameters } from '@vtmn/showcase-core/csf/components/overlays/tooltip.csf';
```

Comment:
Here it's okay, you're well using the correct file: https://github.com/Decathlon/vitamin-web/blob/main/packages/showcases/core/csf/components/overlays/tooltip.csf.js